.github/workflows/test-rust.yml

```diff
@@ -112,3 +112,116 @@ jobs:
 
       - name: Run cargo hack
         run: cargo hack check -p polars --each-feature --no-dev-deps
+
+  build-other-targets:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+        - name: Wasm
+          target: wasm32-unknown-unknown
+          features: >
+            -F abs
```

Comment:
Not that familiar with this. What does -F mean in this context?

Reply:
this is used further down on line 226 — it specifies a feature:

```sh
cargo build --target ${{ matrix.target }} -p polars --no-default-features ${{ matrix.features }}
```